L1Trigger/L1TGlobal/interface/ConditionEvaluation.h

```diff
@@ -202,6 +210,39 @@ namespace l1t {
     }
   }
 
+  // check if a value is greater than a cut or
+  // greater-or-equal depending on the value of the condGEqValue flag
+  // made for AXOL1TL condition to compare cut to score
+  template <class Type1, class Type2>
+  const bool ConditionEvaluation::checkCut(const Type1& cutL, const Type2& value, const bool condGEqValue) const {
+    if (value > 0) {
+      LogTrace("L1GlobalTrigger") << "  checkCut check for condGEqValue = " << condGEqValue << "\n    hex: " << std::hex
+                                  << "cut = " << cutL << " value = " << value << "\n    dec: " << std::dec
+                                  << "cut = " << cutL << " value = " << value << std::endl;
+    }
+
+    if (condGEqValue) {
+      if (value >= (Type2)cutL) {
+        //LogTrace("L1GlobalTrigger") << "    condGEqValue: value >= cut"
+        //    << std::endl;
```

Comment:
Please remove commented code. The if statement can also be reduced down to one line once that's done by the look of it